tasks/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+"""Consolidates invoke tasks and utilities
+
+"""
+from .tasks import *  # noqa
```

Comment:
I'm not sure of the side effects of having both `tasks.py` and `tasks/__init__.py`. It might make more sense to just move all the code from tasks.py into `tasks/__init__.py` to avoid potential clashes.